packages/common/react-components/src/components/index.ts

```diff
@@ -26,4 +26,5 @@ export * from './Tooltip';
 export * from './ThemeProvider';
 export * from './TreeView';
 export * from './Selector';
+export * from './Select';
```

Comment:
Can we delete Selector with this PR?

Reply:
yeah I think i'll replace the other usages first